packages/lwc-language-server/src/javascript/type-mapping.ts

```diff
@@ -217,17 +217,25 @@ function getMemberProperty(propertyObj: ClassProperty): InternalClassMember | nu
         propertyObj.dataProperty,
     );
 
+    // Note there can only be a getter or only a setter, both are not required.
+    // Use the getter if available, or the setter if there is no getter.  
+    let loc: SourceLocation;
+    if (propertyObj.propertyType === 'accessor') {
+        if (propertyObj.getter) {
+            loc = propertyObj.getter.location
+        } else if (propertyObj.setter) {
+            loc = propertyObj.setter.location
+        }
```

Comment:
From local testing it appears there will be no property object without either a getter, setter, both, or a dataProperty.  The`externalToInternalLoc` method handle the case where this value is empty.